src/bit_array.cr

```diff
@@ -226,6 +226,13 @@ struct BitArray
     Slice.new(@bits.as(Pointer(UInt8)), (@size / 8.0).ceil.to_i)
   end
 
+  # See `Object#hash(hasher)`
+  def hash(hasher)
+    hasher = size.hash(hasher)
+    hasher = to_slice.hash(hasher)
```

Comment:
I think you should edit the `NOTE` above in `==` to mention that `hash` would have to change too.

Reply:
Thanks, done.